### PR TITLE
Tune model assumptions

### PR DESCRIPTION
**Pull request by willcattoneeca**
This PR brings in some further improvements to the model configuration prior to the public launch scheduled for this Weds.

* Updates the hot water model to account for outdoor cylinder location (for HWHP systems) and indoor location (for other cylinder types) and refines the efficiencies for natural gas hot water systems. Corrects wood thrown density and calorific value figures to match those in the reference.

* Pre-filters plans to those from the big-four gentailers, reducing the risk of plans belonging to bundles e.g. with internet being mistakenly selected by the optimization routines. Modifies the gas-using household profile to not have gas space heating as this is uncommon in NZ. Updates the electricity and natural gas plans selected as a result.

* Updates the lookup tables based on these new model assumptions.

* Tests passing.

### CODE REVIEW

File: tests/services/test_helpers.py [hunk 1]
```diff
@@ -73,24 +73,39 @@ def test_standing_loss_kwh_per_year():
     """
     Test the standing_loss_kwh_per_year hot water energy use function.
     """
+    climate_zone = "Wellington"
```
Comment (gg-eeca) [inline tests/services/test_helpers.py:76]: Checked there all match spreadsheet (with some rounding error).

**Review by gg-eeca — APPROVED** [review]
Some changes suggested, but will currently behave as intended :-)

File: app/services/helpers.py [hunk 2]
```diff
@@ -254,28 +263,37 @@ def hot_water_heating_efficiency(hot_water_heating_source, climate_zone):
     raise ValueError(f"Unknown hot water heating source: {hot_water_heating_source}")
 
 
-def hot_water_cylinder_heat_loss_kwh_per_day(tank_size):
+def hot_water_cylinder_heat_loss_kwh_per_day(tank_size, delta_t=47):
```
[the remark this answers is not in the document]
Comment (willcattoneeca) [reply app/services/helpers.py]: Good idea - that's cleaner.

File: tests/services/test_helpers.py (hunk 1 above)
Comment (willcattoneeca) [reply tests/services/test_helpers.py:76]: Thanks!

**Comment by willcattoneeca** [comment]
Happy to merge with gg-eeca's review.

**Pull request merged**